basic_OOP/OOP.php

```diff
@@ -0,0 +1,109 @@
+<?php 
+
+    //----------------------------------------Homework1----------------------------------
+    abstract class Shape {
+        protected $width;
+        protected $height;
+
+        public function __construct($width, $height) {
+            $this->width = $width;
+            $this->height = $height;
+        }
+        protected function area() {

```

Comment:
Why don't we use abstract function?
https://www.w3schools.com/php/php_oop_classes_abstract.asp

Reply:
I have changed this to abstract function
Thanks for your comment

Reply:
you should write like this
```
abstract protected function area();
```

Reply:
yes, i have changed it
Thanks for your comment